doc/bitmasks.rst

```diff
@@ -2,4 +2,71 @@
 Bit Masks in DESI
 =================
 
-This page describest bit masks in DESI.
+This page links to the URLs defining the bitmasks found in DESI files.
+
+Redshift fitting (Redrock) masks
+
+=================================== ==================
+BIT_MASK                            URL
+=================================== ==================
+ZWARN		                        `ZWARN`_
```

Comment:
It looks like you've got TAB characters instead of spaces.  I know this is fussy, but spaces will be better supported long-term.